src/iter/ordered_iter.rs

```diff
@@ -243,29 +252,43 @@ impl<K: Ord+Eq,
     }
 }
 
-impl<'a, K: Ord> OrderedSetIterator<&'a K> for btree_set::Iter<'a, K> {}
-impl<'a, K: Ord, V> OrderedMapIterator<&'a K, &'a V> for btree_map::Iter<'a, K, V> {}
+impl<'a, K: Ord> OrderedSetIterator<Key=&'a K> for btree_set::Iter<'a, K> {}
```

Comment:
These lines use the wrong syntax for associated types, and anyway, `OrderedSetIterator` no longer has an associated type (it uses `Iterator::Item`), per the changes above. Lines 255-256, for example, should be changed to
```rust
impl<'a, K: Ord> OrderedSetIterator for btree_set::Iter<'a, K> {}

impl<'a, K: Ord, V> OrderedMapIterator for btree_map::Iter<'a, K, V> {
    type Key = &'a K;
    type Val = &'a V;
}
```